UI Cleanup for FullContact Plugin

tyjoll96 opened the pull request:
Information formatted and given data checks.

Readme updated with additional steps.

tyjoll96 committed:
UI Cleanup for FullContact Plugin

Information formatted and given data checks.

Readme updated with additional steps.

markcallen commented on README.md:
```diff
@@ -86,3 +71,40 @@ Open Hootsuite https://hootsuite.com/dashboard
+
+When on the dashboard, you will see your API key in the lower section of the page.
+
+## Creating Config File
```
You don't need to document this, you need to implement it in the code.

markcallen commented on README.md:
`````diff
@@ -86,3 +71,40 @@ Open Hootsuite https://hootsuite.com/dashboard
+mkdir config
+````
+
+create the development version:
`````
This file should be created by default with the necessary parameter with no value.

markcallen commented on README.md:
`````diff
@@ -86,3 +71,40 @@ Open Hootsuite https://hootsuite.com/dashboard
+vi config/production.json
+````
+
+Getting FullContact API key in your code:
`````
This should be in the code, not in the doc.

markcallen reviewed (commented):
Couple of things to clean up.

tyjoll96 committed:
Clean Up

Added icons and documentation edits for adding images during app creation.

HTML files renamed.

Chevron added for social network toggle.

tyjoll96 committed:
Updated README and UI

Readme docker section changed for docker-compose.

Pop up made scrollable.
JSON pretty printed.

markcallen reviewed (approved):
:+1:
